en/docs/install-and-setup/setup/mi-setup/deployment/mi-cicd-vm.md

```diff
@@ -0,0 +1,150 @@
+[![VM based CI/CD for Micro Integrator]({{base_path}}/assets/img/deploy/mi-cicd-VM.png)]({{base_path}}/assets/img/deploy/mi-cicd-VM.png)
+
+#### Overview
+
+There are two kinds of jobs in Jenkins. One kind is for the Integration projects. Another kind is for the Descriptor repositories. 
+
+*   We need to maintain one jenkins job per Integration Project repo.
+*   The Integration Project has to be a Maven Multi Module project and it has to contain one Composite Exporter module. By default, the script supports one Exporter module. Users can customize it to support multiple Exporter modules per Integration Project.
```

Comment:
Shall we link to the description of MMM project?

Reply:
Done